conans/test/functional/generators/deploy_test.py

```diff
@@ -217,3 +218,37 @@ def test_existing_file_symbolic_links(self):
         self.assertFalse(os.path.islink(header_path))
         linkto = os.path.join(os.path.dirname(link_path), os.readlink(link_path))
         self.assertEqual(linkto, header_path)
+
+
+@unittest.skipIf(platform.system() == "Windows", "Permissions in NIX systems only")
+class DeployGeneratorSymbolicLinkFolderTest(unittest.TestCase):
+
+    def setUp(self):
+        conanfile = textwrap.dedent("""
```

Comment:
We typically indent the content inside the `textwrap.dedent`

Reply:
done!